### ISSUE TITLE
Use Grafana styles in the Saga website

### ISSUE THREAD

**Issue by Clarity-89** [grafana/grafana#77376]

**Comment by JoaoSilvaGrafana** [comment]
Findings from checking this:
- Switch: font-size is set to 16px, this affects the size of the box surrounding the switch
- Checkbox: Can't see the issue locally
- TextArea: Font-family is set to monospace, need to figure out how to override that in Grafana
- Slider: Missing react-slider styles

**Comment by Clarity-89** [comment]
> Findings from checking this:
> 
This was fixed by https://github.com/grafana/design-system/pull/98, forgot to update the issue.
> 
> I think ideally the styles applied on this section would be Grafana styles. Not sure there is an easy way to achieve this. Maybe we setup all these templates on storybook and iframe from there? This would mean keeping the code in two separate places, which is not ideal. What do you think?

There's a section about this with pros and cons (Proposal 1) in the [Design doc](https://docs.google.com/document/d/12VdeFTxWapU6kv2NJNabMV41Y8jACTTKHliFOfQQPyQ/edit).

**Comment by JoaoSilvaGrafana** [comment]
> There's a section about this with pros and cons (Proposal 1) in the [Design doc](https://docs.google.com/document/d/12VdeFTxWapU6kv2NJNabMV41Y8jACTTKHliFOfQQPyQ/edit).

Ah yes forgot I read that somewhere 😅  I do think the preference would be to keep all of the templates code just on Saga. Our alternative to keep the stylings would be to create some sort of component in Saga that uses the `ThemeContext` and loads the Grafana theme.

**Comment by Clarity-89** [comment]
Yeah, I agree. If we've decided to move away from using Storybook for the design system stuff then it'd make sense to keep templates in Saga. Also I'm not sure how much setup the webpack's raw loader needs to make it work in Storybook.

I think the main issue with the Saga currently is that the font looks different from Grafana in some cases, once that's fixed, the UI will be mostly consistent.

**Comment by JoaoSilvaGrafana** [comment]
> Yeah, I agree. If we've decided to move away from using Storybook for the design system stuff then it'd make sense to keep templates in Saga. Also I'm not sure how much setup the webpack's raw loader needs to make it work in Storybook.
> 
> I think the main issue with the Saga currently is that the font looks different from Grafana in some cases, once that's fixed, the UI will be mostly consistent.

If that is the main issue, we _could_ just override the font differences. I know one key thing is that we use 16px font size in Saga and Grafana is 14px. If we wrap all the templates in some styling that overrides that maybe it would work?

**Comment by Clarity-89** [comment]
> > Yeah, I agree. If we've decided to move away from using Storybook for the design system stuff then it'd make sense to keep templates in Saga. Also I'm not sure how much setup the webpack's raw loader needs to make it work in Storybook.
> > I think the main issue with the Saga currently is that the font looks different from Grafana in some cases, once that's fixed, the UI will be mostly consistent.
> 
> If that is the main issue, we _could_ just override the font differences. I know one key thing is that we use 16px font size in Saga and Grafana is 14px. If we wrap all the templates in some styling that overrides that maybe it would work?

Yeah, that's one of the issues and I think wrapping could work here, I just wasn't able to find a quick solution for that. Another issue is that the medium bold font (font-weight: 500) doesn't seem to be available in the Saga website for some reason.

**Comment by Clarity-89** [comment]
All the issues were fixed.
